src/Temporalio/Worker/TemporalWorkerOptions.cs

```diff
@@ -207,9 +207,11 @@ public TemporalWorkerOptions()
 
         /// <summary>
         /// Gets or sets the maximum number of concurrent poll workflow task requests we will
-        /// perform at a time on this worker's task queue. Default is 5.
+        /// perform at a time on this worker's task queue.
         /// </summary>
-        public int MaxConcurrentWorkflowTaskPolls { get; set; } = 5;
+        /// <remarks>If set, will override any value set in <see cref="WorkflowTaskPollerBehavior"/>.</remarks>
+        [Obsolete("Use WorkflowTaskPollerBehavior instead")]
+        public int? MaxConcurrentWorkflowTaskPolls { get; set; }
```

Comment:
Hrmm, not sure it is deprecated and would have to think through whether changing this from `int` to `int?` is a backwards compatible change we're ok with (it's a bit different to change a data type on an options set than it is in a parameter list). Will bring up internally.

One option is to leave this alone and make `PollerBehavior` the optional item and say if _that_ is set it overrides _this_. That is usually a safer way to introduce something new and opt-in. Can even have a check if you want that confirms if they changed the int field from the default, then the poller behavior should be null.

Reply:
Ok, confirmed with @Sushisource this doesn't break expectations. So lets leave these properties alone (no deprecation, no data type change) and just have poller behavior presence _override_ these if set (can note this in docs above both). We don't have to go the extra step of throwing an exception if thee are changed from default _and_ poller behavior present, but can if you think it's worth it.